README.md

```diff
@@ -4,16 +4,16 @@ SonarLint IntelliJ Plugin
 [![Build Status](https://api.cirrus-ci.com/github/SonarSource/sonarlint-intellij.svg?branch=master)](https://cirrus-ci.com/github/SonarSource/sonarlint-intellij)
 [![Quality Gate](https://next.sonarqube.com/sonarqube/api/project_badges/measure?project=org.sonarsource.sonarlint.intellij%3Asonarlint-intellij&metric=alert_status)](https://next.sonarqube.com/sonarqube/dashboard?id=org.sonarsource.sonarlint.intellij%3Asonarlint-intellij)
 
-SonarLint is an IDE extension that helps you detect and fix quality issues as you write code.
+SonarLint is an IDE extension that helps you detect and fix quality issues as you write [Clean Code](https://www.sonarsource.com/solutions/clean-code/).
```

Comment:
I have 2 comments on this line:

> helps you detect and fix quality issues as you write Clean Code

I find the phrasing a bit imprecise, I understand it as if people are already writing clean code, not that we help them achieve that. Maybe something like: 

> SonarLint is an IDE extension that supports you in writing Clean Code, by detecting and helping you fix quality issues as you type

My second point is that READMEs are usually read by technical people, so I'm a bit reluctant to point to a marketing website from here, so I would prefer to point to https://docs.sonarsource.com/sonarlint/intellij/concepts/clean-code/